test/simplifyExpression/basicsSearch/basicTest.js

```diff
@@ -87,7 +85,7 @@ describe('basic rules', function() {
       ['0 + x', 'x'],
       ['0 + (x + 1)', 'x + 1'],
     ];
-    tests.forEach(t => testSimplify(t[0], t[1], basic_rules.removeAdditionOfZero));
+    tests.forEach(t => testSimplify(t[0], t[1], basics.REMOVE_ADDING_ZERO, ChangeTypes.REMOVE_ADDING_ZERO, basics.REMOVE_ADDING_ZERO_REVERSE));
```

Comment:
I find it weird that you're passing in the reverse - how soon can you make the change in math-rules? can you do it before finishing this PR?

otherwise I'd rather you pass in an array than tacking a new argument at the end - it's kinda hard to know what each argument is